requirements.txt

```diff
@@ -1,7 +1,6 @@
 numpy~=1.15
 periodictable~=1.5
-picmistandard==0.0.13
-scipy~=1.5  # picmistandard bug: https://github.com/picmi-standard/picmi/pull/34
+picmistandard==0.0.14
```

Comment:
Do we need to merge and release https://github.com/picmi-standard/picmi/pull/35 first?

Reply:
Yes, the picmi PR needs to be merged first and then uploaded to PyPI.

Reply:
Perfect, merged now and ready for release to PyPI

Reply:
picmi is updated on PyPI, rerunning CI tests